cmd/aem/root.go

```diff
@@ -29,6 +29,7 @@ func (c *CLI) rootCmd() *cobra.Command {
 	cmd.AddCommand(c.cryptoCmd())
 	cmd.AddCommand(c.sslCmd())
 	cmd.AddCommand(c.gtsCmd())
+	cmd.AddCommand(c.userCmd())
```

Comment:
I prefer to organize it more future to be more future-ready

root -> auth -> user
                     -> group